subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -61,7 +64,12 @@ from the root project directory and let Gradle do its thing.
 That basically consists of parsing the existing POMs and generating the corresponding Gradle build scripts.
 Gradle will also create a settings script if you're migrating a <<multi_project_builds#multi_project_builds,multi-project build>>.
 
-You'll find that the new Gradle build includes any custom repositories specified in the POM, your external and inter-project dependencies, the appropriate plugins (any of `maven`, `java`, and `war`), and more.
+You'll find that the new Gradle build includes the following:
+
+ * All the custom repositories that are specified in the POM
+ * Your external and inter-project dependencies
+ * The appropriate plugins to build the project (limited to one or more of the <<maven_plugin#maven_plugin,Maven>>, <<java_plugin#java_plugin,Java>> and <<war_plugin#war_plugin,War>> Plugins)
```

Comment:
The `init` here is kind of outdated from my point of view.
- Instead of a Groovy build script, it should create a Kotlin build script (or present the user with a choice)
- Instead of `maven` in should apply `maven-publish`

I am not sure if we want to invest into doing these changes.

I wonder if we should add notes here for now. Saying something like:

_Note: The init plugin creates a Groovy build script. with Gradle 5.0+, we recommend to use Kotlin build scripts instead for a better IDE experience (link to Groovy to Kotlin migration - do we have that?)._

_Note: It is recommended to change the `maven` to the `maven-publish` plugin (link to userguide/publishing_maven.html)_

Reply:
I did raise [an issue](https://github.com/gradle/gradle/issues/7952) regarding the Maven Publish Plugin.

I should probably raise one about `--dsl kotlin` not working with `--type pom`.

Reply:
I think migrating to Kotlin DSL is asking a bit too much of the user when they should be focused on the other aspects of the migration. And it just highlights even more strongly how out of date the `init` task is. We should instead try to give `init` some attention if we want to support folks migrating from Maven.

Reply:
:+1: I didn't know about the issue and that work on it is already planned. Please raise an issue about --dsl kotlin not working with --type pom as well. Maybe @marcphilipp can plan that in as well.

Then we don't need to change anything in this guide.